src/client/components/TimezonePicker.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
```

Comment:
You should move this into the inputs file: https://github.com/EdgeApp/edge-logs-server/blob/master/src/client/components/Inputs.tsx

Then you would notice you can reuse the styles there.

Reply:
As requested, this component was moved to the inputs file. TimezonePicker.tsx will be deleted in fixup commit.